### ISSUE TITLE
Loading timeout / Blank page

### ISSUE THREAD

**Issue by rzschoch** [brocessing/kirby-webpack#40]
Hi,

the initial setup only displays a blank page and the browser timeouts while loading. The only way I found is to use my own local proxy (via proxy setting in main.config.js).

Is there anything I could try?

System: Mac OS 10.13.4

Thanks!

**Comment by pqml** [comment]
Indeed, there is an issue with the new 2.5.10 version of `kirby` and `panel`. I tried with 2.5.9 en everything is working fine. 

I'll look for a fix

**Comment by pqml** [comment]
I succeed to reproduce the issue once, but now it seems to work fine. Have you tried to re-start the dev-server ?

**Comment by rzschoch** [comment]
Yes, I re-started the dev server multiple times, used Chrome, Firefox, Safari and also tried different preprocessor settings (sass and less) without any change.

Any idea how I could debug the issue and provide you with helpful messages?

**Comment by rzschoch** [comment]
I also tried to use it width older versions of Kirby (2.5.9 and 2.5.8) – unfortunately that didn’t help 😞

**Comment by pqml** [comment]
What is your php cli version ? 

Maybe it's also something related to our use of IPv6 to get the php server properly working on localhost on MacOS (see http://php.net/manual/fr/features.commandline.webserver.php#120449)
Do you have any particular network setup ?

**Comment by rzschoch** [comment]
Thanks for your time.

My php version: PHP 7.1.14 (cli) (built: Feb  7 2018 18:33:30) ( NTS )

When I start a php server manually (php -S localhost:8000) and change the proxy setting to localhost:8000 everything works. The server log then shows that localhost resolves to 127.0.0.1.

Hope this helps?

**Comment by obelmont** [comment]
Hi, I seem to be having a similar issue to this one, oddly enough this only occurs when I attempt to access the panel.

My php version: PHP 7.2.4 (cli) (built: Mar 29 2018 15:32:43) ( NTS )

Haven't been able to get the dev server working through the proxy settings. However the panel is working when I access it directly through the manual php server (php -S localhost:8000).

When I downgrade php to 7.0 (PHP 7.0.29 (cli) (built: Apr 12 2018 03:13:39) ( NTS )) it seems to work fine.

Happy to help in any way I can with this one!

**Comment by pqml** [comment]
It's a really strange behaviour and I have no issues using PHP 7.2.4 cli and macOS 10.13.4

With the latest kirby-webpack version you can try to change `phpHost: 'localhost'` to `phpHost: '127.0.0.1'` in `main.config.js`

You will also have to rename `www/site/config/config.localhost.php` to `www/site/config/config.127.0.0.1.php`

**Comment by obelmont** [comment]
I think my issue is actually entirely PHP install related. I am unsure if this is the same for @rzschoch.

I tried manually starting a php server and heading to the panel where I refreshed the page a few times and the php server had a segmentation fault and shut down. This only happens on php 7.2 and not 7.0. I'll post here again if I can confirm that this is the case!

**Comment by PeteCrighton** [comment]
> With the latest kirby-webpack version you can try to change `phpHost: 'localhost'` to `phpHost: '127.0.0.1'` in `main.config.js`
>
> You will also have to rename `www/site/config/config.localhost.php` to `www/site/config/config.127.0.0.1.php`

I think I had the same issue as @rzschoch (on Linux), and this fixed it for me.

Should this `config.127.0.0.1.php` be prevented from being deployed to the production server?

**Comment by pqml** [comment]
Interesting. On the next release I will set 127.0.0.1 as the default host instead of localhost since it seems to fix a couple of issues

I don't think you need to prevent the file from being deployed. It should not be used by your server - see https://getkirby.com/docs/developer-guide/configuration/options
